packages/devtools_app/lib/src/ui/analytics.dart

```diff
@@ -153,17 +157,181 @@ class GtagExceptionDevTools extends GtagException {
   external String get ide_launched;
 }
 
+/// Request Flutter tool stored value enabled (GA enabled) stored in ~\.flutter.
+/// Return bool or null, null implies Flutter Tool has never been run.
+Future<bool> get isFlutterGAEnabled async {
+  bool enabled = false;
+
+  var resp =
+      await HttpRequest.request('/api/getFlutterGAEnabled', method: 'POST');
+  if (resp.status == HttpStatus.ok) {
+    // Return value 'null' implies Flutter tool has never been run return null.
+    if (resp.responseText == 'null') return null;
+
+    enabled = resp.responseText == 'true';
+  } else {
+    log('HttpRequest api/getFlutterGAEnabled failed status = ${resp.status}',
+        LogLevel.error);
+  }
+  return enabled;
+}
+
+/// Request Flutter tool stored clientID (GA enabled) stored in ~\.flutter.
+/// Return clientId as a String or null, null implies Flutter Tool has never
+/// been run.
+Future<String> flutterGAClientID() async {
+  String clientId = '';
+
+  var resp =
+      await HttpRequest.request('/api/getFlutterGAClientId', method: 'POST');
+  if (resp.status == HttpStatus.ok) {
+    // Return value 'null' implies Flutter tool has never been run return null.
+    if (resp.responseText == 'null') return null;
+
+    clientId = resp.responseText;
+  } else {
+    log('HttpRequest api/getFlutterGAClientId failed status = ${resp.status}',
+        LogLevel.error);
+  }
+  return clientId;
+}
+
+/// Request all .devtools properties resets to their default values.
+void resetDevTools() async {
+  final resp = await HttpRequest.request('api/resetDevTools', method: 'POST');
+  if (resp.status == HttpStatus.ok) {
+    assert(resp.responseText == 'true');
+  } else {
+    log(
+      'HttpRequest api/resetDevTools failed status = ${resp.status}',
```

Comment:
same here - create const for `api/resetDevTools`

Reply:
Done